### DISCUSSION TITLE
Help? Removing black background parts, Retaining black text

### DISCUSSION THREAD

**Discussion by tg01** [ImageMagick/ImageMagick#1953] [General]
Hi Guys and Gals
Could you help?
I've an image that I need to remove the black background parts from whilst retaining the black text to run through Tesseract OCR.
Having cropped, transformed, trim'd, median'd, got the image to black and while, then autolevel'd I have this image:
![82](https://user-images.githubusercontent.com/4324191/80579645-4c20b100-8a02-11ea-86df-babffab498f8.jpg)
I need to remove the black angled background.

(for reference this was the original graphic:
![82-original](https://user-images.githubusercontent.com/4324191/80579842-94d86a00-8a02-11ea-8945-d6ebe7ccf198.jpg)
)

Thank you

**Comment by tg01** [comment]
I need to get to this:
![82-nobg](https://user-images.githubusercontent.com/4324191/80580034-d36e2480-8a02-11ea-9cc8-5f6d877f8c19.jpg)
Using imagick (php imagemagick)

**Comment by tg01** [comment]
Spent a while longer on it and got a solution.
With each any every image I am processing the angles are roughly the same for each.
By creating a PNG overlay image, using resize image and compositeImage I was able to place a white cover on the image each time.

**Reply by dlemstra** [reply]
Could you include the command that you executed and the overlay image here for a future reader?

**Comment by tg01** [comment]
That's exactly what I did.
Though the ocr is still struggling.
May enlarge the file and smoothen the letters to get the result.
